src/centos/scripts/start-marklogic.sh

```diff
@@ -87,8 +87,8 @@ fi
 # Install Converters if required
 ################################################################
 if [[ "${INSTALL_CONVERTERS}" == "true" ]]; then
-    if [[ -d "/opt/MarkLogic/Converters" ]]; then
-        info "Converters directory: /opt/MarkLogic/Converters already exists, skipping installation of converters."
+    if [[ -d "opt/MarkLogic/Converters" ]]; then
```

Comment:
Why remove the leading slash? I would prefer an absolute path rather than relative and assume running in the root dir.

Reply:
Mistake on my end, good catch. Just fixed it.